### ISSUE TITLE
FTL/dnsmasq port

### ISSUE THREAD

**Issue by brccabral** [pi-hole/docker-pi-hole#1221]
I have started pihole using this docker-compose.yml

```yml
version: "3"

# More info at https://github.com/pi-hole/docker-pi-hole/ and https://docs.pi-hole.net/
services:
  pihole:
    container_name: pihole
    image: pihole/pihole:latest
    # For DHCP it is recommended to remove these ports and instead add: network_mode: "host"
    network_mode: "host"
    environment:
      TZ: 'America/Los_Angeles'
      WEBPASSWORD: XXXXXX
    # Volumes store your data between container upgrades
    volumes:
      - './etc-pihole:/etc/pihole'
      - './etc-dnsmasq.d:/etc/dnsmasq.d'
      - './var-log-pihole:/var/log/pihole'
    #   https://github.com/pi-hole/docker-pi-hole#note-on-capabilities
    cap_add:
      - NET_ADMIN # Required if you are using Pi-hole as your DHCP server, else not needed
    restart: unless-stopped

```

`FTL.log` shows many messages as 
```log
[2022-10-04 17:43:28.557 6828M] FATAL ERROR in dnsmasq core: failed to create listening socket for port 53: Address already in use
```

Only after editing `/etc/dnsmasq.conf` by adding another port FTL was able to start
```console
# cat /etc/dnsmasq.conf
conf-dir=/etc/dnsmasq.d

user=pihole

port=5353
```

What is the other process that is listening to port 53? Should this be configured by default?

**Comment by dschaper** [comment]
`   network_mode: "host"`

That puts the container directly on the wire so any DNS server running on the host will conflict with Pi-hole.

>What is the other process that is listening to port 53? Should this be configured by default?

That's something you'll need to check on the host server. Changing Pi-hole to be on port 5353 won't do much to help you unless you can change every client to use that port. 

What operating system are you running on the host server?

**Comment by brccabral** [comment]
If I remove `network_mode` and set the ports docker-compose fails
```yml
    # For DHCP it is recommended to remove these ports and instead add: network_mode: "host"
    #network_mode: "host"
    ports:
      - "53:53/tcp"
      - "53:53/udp"
      #- "67:67/udp" # Only required if you are using Pi-hole as your DHCP server
      - "80:80/tcp"
```
`Error response from daemon: driver failed programming external connectivity on endpoint pihole (a2ba63d84e454e55ad836871c78ed58a0b068663ac88dbcf4e79fb93a3098ff3): Error starting userland proxy: listen tcp4 0.0.0.0:53: bind: address already in use`

I am using Manjaro

**Comment by brccabral** [comment]
ok, my distribution (based on Manjaro) had a pihole-FTL installed by default, I didn't know it, I'm just starting using it
